app/src/main/kotlin/com/intive/picover/common/platform/FirebaseModule.kt

```diff
@@ -0,0 +1,17 @@
+package com.intive.picover.common.platform
+
+import com.google.firebase.storage.FirebaseStorage
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object FirebaseModule {
```

Comment:
In my opinion, we should start with some global `AppModule` for common singleton dependencies, to not create a few files with only a single provider.